katdal/visdatav4.py

```diff
@@ -78,11 +78,36 @@ def _add_sensor_alias(cache, new_name, old_name):
         pass
 
 
+def _normalise_cal_products(products, cal_streams):
+    """"""
+    requested_cal_products = _selection_to_list(products, all=cal_streams,
+                                                default=DEFAULT_CAL_PRODUCTS)
+    skip_missing_products = products in ('all', 'default') or any(
+        ['.' not in product for product in requested_cal_products])
```

Comment:
The square brackets aren't needed: `any` works fine on generator expressions.

Reply:
Oops, yes.